src/deskstar-backend/Teststar.Tests/Tests/AuthUsecasesTests.cs

```diff
@@ -9,151 +9,145 @@
+        using var mogDB = new DataContext();
+        var company = new Company
         {
-            var company = new Company
-            {
-                CompanyId = new Guid(),
-                CompanyName = "gehmalbierholn"
-            };
-
-            mogDB.Companies.Add(company);
-            mogDB.SaveChanges();
-
-            //arrange
-            var logger = new Mock<ILogger<AuthUsecases>>();
-            var subject = new AuthUsecases(logger.Object, mogDB);
+            CompanyId = new Guid(),
+            CompanyName = "gehmalbierholn"
+        };
+
+        mogDB.Companies.Add(company);
+        mogDB.SaveChanges();
+
+        //arrange
+        var logger = new Mock<ILogger<AuthUsecases>>();
+        var subject = new AuthUsecases(logger.Object, mogDB);
            
-            var user = new RegisterUser();
-            user.MailAddress = "test@mail.de";
-            user.FirstName = "testF";
-            user.LastName = "testL";
-            user.Password = "testpw";
-            user.CompanyId = company.CompanyId;
-
-            //act
-            var result = subject.registerUser(user);
-
-            //assert
-            Assert.That(result);
-        }
+        var user = new RegisterUser();
+        user.MailAddress = "test@mail.de";
+        user.FirstName = "testF";
+        user.LastName = "testL";
+        user.Password = "testpw";
+        user.CompanyId = company.CompanyId;
+
+        //act
+        var result = subject.RegisterUser(user);
+
+        //assert
+        Assert.That(result, Is.EqualTo(RegisterReturn.Ok));
     }
 
     [Test]
     public void RegisterUser_ExistingMail()
     {
-        using (var mogDB = new DataContext())
-        {
-            AddOneCompany_AddOneUser(mogDB, new PasswordHasher<User>());
+        using var mogDB = new DataContext();
+        AddOneCompany_AddOneUser(mogDB, new PasswordHasher<User>());
 
-            //arrange
-            var logger = new Mock<ILogger<AuthUsecases>>();
-            var subject = new AuthUsecases(logger.Object, mogDB);
-            var hasher = new PasswordHasher<User>();
+        //arrange
+        var logger = new Mock<ILogger<AuthUsecases>>();
+        var subject = new AuthUsecases(logger.Object, mogDB);
+        var hasher = new PasswordHasher<User>();
 
-            var user = new RegisterUser();
-            user.MailAddress = "test@mail.de";
-            user.FirstName = "testF";
-            user.LastName = "testL";
-            user.Password = "password";
+        var user = new RegisterUser
+        {
+            MailAddress = "test@mail.de",
+            FirstName = "testF",
+            LastName = "testL",
+            Password = "password"
+        };
 
-            //act
-            var result = subject.registerUser(user);
+        //act
+        var result = subject.RegisterUser(user);
 
-            //assert
-            Assert.That(result, Is.False);
-        }
+        //assert
+        Assert.That(result, Is.False);
```

Comment:
We should check for RegisterReturn.MailAddressInUse here. Either way... the method return a .CompanyNotFound because the user to be registered is missing the company id field.